src/adagrams.js

```diff
@@ -1,15 +1,114 @@
-export const drawLetters = () => {
-  // Implement this method for wave 1
-};
+class Adagrams {
+  static drawLetters() {
+    const letterDict = {
```

Comment:
Even within a class, `letterDict` could be moved out into the main scope, so you don't recreate the object every time `drawLetters` is called.